types/config.go

```diff
@@ -225,3 +208,14 @@ func KeyringServiceName() string {
 	}
 	return version.Name
 }
+
+// Optional: expose sealed config with timeout
+func GetSealedConfig(ctx context.Context) (*Config, error) {
```

Comment:
whats the use case of this?

Reply:
i dont actually know - this logic exists in the previous version - the diff just moved it a bunch